sdk/src/main/scala/io/horizen/account/mempool/MempoolMap.scala

```diff
@@ -59,15 +60,21 @@ class MempoolMap(
     if (!contains(ethTransaction.id)) {
       val account = ethTransaction.getFrom
 
-      //Reject transactions that are too big
+      // Reject transactions that are too big
       val txSize = ethTransaction.size()
       if (txSize > MaxTxSize) {
         log.trace(s"Transaction $ethTransaction size exceeds maximum allowed size: current size $txSize, " +
           s"maximum size: $MaxTxSize")
         throw TxOversizedException(account.asInstanceOf[AddressProposition].address(), txSize)
       }
 
-      //Reject transactions with a nonce gap too big
+      // Reject transaction with max fee less than the current minimum base fee
+      val currentEpochNumber = baseStateReaderProvider.getBaseStateReader().getConsensusEpochNumber.getOrElse(0)
+      if (GasFeeFork.get(currentEpochNumber).baseFeeMinimum.compareTo(ethTransaction.getMaxFeePerGas) > 0) {
```

Comment:
can you double check that with old Legacy (Type 0) txs the check is working properly? (maybe add a unit test for that)